react/components/SearchResultContainer.js

```diff
@@ -132,9 +98,7 @@ const SearchResultContainer = props => {
         <ResultComponent
           {...props}
           showMore={showMore}
-          breadcrumbsProps={getBreadcrumbsProps(
-            Object.assign({}, params, { categoriesTrees, loading })
-          )}
+          breadcrumbsProps={{ categoryTree: breadcrumb }}
```

Comment:
The name `categoryTree` still makes sense?

Reply:
Não quero causar breaking change ou ficar criando mais uma prop...